### PR TITLE
Move to version 2.5.1

### PR DESCRIPTION
**Pull request by madhuthorat**
Signed-off-by: Madhu Thorat <madhu.punjabi@in.ibm.com>

## Pull request checklist
Changes to move to 2.5.1

```

## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature Enhancement
- [ ] Test Automation 
- [ ] Code Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] Community Operator listing 
- [ ] Other (please describe): 


## What is the current behavior?
NA

## What is the new behavior?
NA

## How risky is this change?
- [x] Small, isolated change
- [ ] Medium, requires regression testing
- [ ] Large, requires functional and regression testing

### CODE REVIEW

File: operator/controllers/config/constants.go [hunk 1]
```diff
@@ -77,9 +77,9 @@ const (
 	AttacherLeaderLivenessPort = int32(8080)
 
 	//  Default images for containers
-	CSIDriverPluginImage        = "quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-driver:v2.5.0"
+	CSIDriverPluginImage        = "quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-driver:v2.5.1"
 	CSINodeDriverRegistrarImage = "us.gcr.io/k8s-artifacts-prod/sig-storage/csi-node-driver-registrar:v2.4.0"
-	LivenessProbeImage          = "us.gcr.io/k8s-artifacts-prod/sig-storage/livenessprobe:v2.6.0"
+	LivenessProbeImage          = "us.gcr.io/k8s-artifacts-prod/sig-storage/livenessprobe:v2.5.1"
```
Comment (Jainbrt) [inline operator/controllers/config/constants.go:82]: I think livenessprobe versions is 2.6.0 only for CSI 2.5.1. too

**Review by Jainbrt — CHANGES_REQUESTED** [review]

File: cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/inventory/ibmCSIScaleOperator/resources.yaml [hunk 2]
```diff
@@ -147,7 +147,7 @@ resources:
     - metadata:
         name: livenessprobe
       image: cp/spectrum/scale/csi/livenessprobe
-      tag: v2.6.0
+      tag: v2.5.1
```
[the remark this answers is not in the document]
Comment (madhuthorat) [reply cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/inventory/ibmCSIScaleOperator/resources.yaml]: done

File: cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/inventory/ibmCSIScaleOperator/resources.yaml [hunk 3]
```diff
@@ -158,19 +158,19 @@ resources:
         platform:
           architecture: amd64
           os: linux
-        tag: v2.6.0-amd64
+        tag: v2.5.1-amd64
```
Comment (Jainbrt) [inline cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/inventory/ibmCSIScaleOperator/resources.yaml:161]: I think livenessprobe versions is 2.6.0 only for CSI 2.5.1. too

File: cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/inventory/ibmCSIScaleOperator/resources.yaml [hunk 4]
```diff
@@ -158,19 +158,19 @@ resources:
...
       - digest: sha256:218dba5e3768e788451a7571d8a53548c69c53e57f06769956171b8f60a911f0
         mediaType: application/vnd.docker.distribution.manifest.v2
         platform:
           architecture: ppc64le
           os: linux
-        tag: v2.6.0-ppc64le
+        tag: v2.5.1-ppc64le
```
Comment (Jainbrt) [inline cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/inventory/ibmCSIScaleOperator/resources.yaml:167]: I think livenessprobe versions is 2.6.0 only for CSI 2.5.1. too

File: cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/inventory/ibmCSIScaleOperator/resources.yaml [hunk 5]
```diff
@@ -158,19 +158,19 @@ resources:
...
       - digest: sha256:a7bfc3daebcc7c500748e48dcfb06d649b7707f0240397f0b8f8611c9648df2e   
         mediaType: application/vnd.docker.distribution.manifest.v2
         platform:
           architecture: s390x
           os: linux
-        tag: v2.6.0-s390x
+        tag: v2.5.1-s390x
```
Comment (Jainbrt) [inline cloudpak/stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/inventory/ibmCSIScaleOperator/resources.yaml:173]: I think livenessprobe versions is 2.6.0 only for CSI 2.5.1. too

**Review by Jainbrt — CHANGES_REQUESTED** [review]

**Review by Jainbrt — CHANGES_REQUESTED** [review]

**Review by Jainbrt — DISMISSED** [review]

**Review by Jainbrt — APPROVED** [review]

**Pull request merged**